upstream_delineator/py/util.py

```diff
@@ -23,6 +23,8 @@
 assert CATCHMENT_PATH
 RIVER_PATH = os.getenv("RIVER_PATH")
 assert RIVER_PATH
+MEGABASINS_PATH = os.getenv("MEGABASINS_PATH")
```

Comment:
what should ppl set this too when running the delineator? 

lmk and i can update this pin messaged https://upstreamtech.slack.com/archives/C07C8RR4610/p1721681020380309?thread_ts=1721680987.163239&cid=C07C8RR4610

Reply:
https://public-hydrology-data.upstream.tech/merit_hydro_vect_level2.shp